src/libs/ReportUtils.ts

```diff
@@ -219,21 +245,25 @@ type OptimisticChatReport = Pick<
     | 'writeCapability'
 >;
 
-type OptimisticTaskReportAction = Pick<
-    ReportAction,
-    | 'actionName'
-    | 'actorAccountID'
-    | 'automatic'
-    | 'avatar'
-    | 'created'
-    | 'isAttachment'
-    | 'message'
-    | 'originalMessage'
-    | 'person'
-    | 'pendingAction'
-    | 'reportActionID'
-    | 'shouldShow'
-    | 'isFirstItem'
+type OptimisticTaskReportAction = {reportActionID: string} & Partial<
+    Pick<
+        ReportAction,
+        | 'actionName'
+        | 'actorAccountID'
+        | 'automatic'
+        | 'avatar'
+        | 'created'
+        | 'isAttachment'
+        | 'message'
+        | 'originalMessage'
+        | 'person'
+        | 'pendingAction'
+        | 'shouldShow'
+        | 'isFirstItem'
+        | 'previousMessage'
+        | 'errors'
+        | 'linkMetadata'
+    >
```

Comment:
Maybe you can turn it back as it was before (without  `{reportActionID: string} & Partial<`) and just use 
`const optimisticReportAction: Partial<ReportUtils.OptimisticTaskReportAction> = {` in this [line](https://github.com/Expensify/App/blob/f82c5941a91c6730514df94b771d50fc204ecd7b/src/libs/actions/Task.ts#L745)